scripts/stubtest/allowlist.txt

```diff
@@ -268,3 +268,9 @@ django.contrib.gis.db.backends.base.features.BaseSpatialFeatures.unsupported_geo
 # Django uses a `*args` parameter but crash if it contains more than 1 element.
 django.http.QueryDict.pop
 django.http.request.QueryDict.pop
+
+# Django incorrectly decorates this with `@classmethod`.
+django.test.selenium.SeleniumTestCase.__init_subclass__
+
+# Django incorrectly uses a signature of `(self, *args, **kwargs)` when it should be `(self)`.
+django.forms.renderers.Jinja2DivFormRenderer.__init__
```

Comment:
This is just wrong in Django, but it will also be gone in Django 6.0 when we move to that in December.